graph/telemetry/istio/istio.go

```diff
@@ -800,6 +836,27 @@ func buildNodeTrafficMap(cluster, namespace string, n graph.Node, o graph.Teleme
 	return trafficMap
 }
 
+// buildCustomLabelsFromConfig build custom addtional labels for prometheus from config
+func buildCustomLabelsFromConfig() string {
+	customLabels := config.Get().ExternalServices.Prometheus.CustomLabels
+	if len(customLabels) == 0 {
+		customLabels = config.Get().ExternalServices.CustomDashboards.Prometheus.CustomLabels
+		if len(customLabels) == 0 {
+			return ""
+		}
```

Comment:
We don't want to do this. Here you are mixing the custom dashboards prometheus config with the regular prometheus config. The custom dashboards prometheus config is supposed to be isolated/separate from the "regular" prometheus config - the point being that the custom dashboards prometheus config points to a completely separate Prometheus instance than that of the Istio Prometheus. So one config should not assume the other - they are separate.

For details, see the kiali documentation:
* https://kiali.io/docs/configuration/custom-dashboard/#using-another-prometheus-instance
* https://kiali.io/docs/configuration/kialis.kiali.io/#.spec.external_services.custom_dashboards.prometheus